src/js/InstitutionStatus.jsx

```diff
@@ -28,7 +28,7 @@ var InstitutionStatus = React.createClass({
     var submission = this.props.institution.currentSubmission;
     var submissionRoute = submission === 0 ? 1 : submission;
     var appLink = '/' + id + '/' + period + '/' + submissionRoute;
-    var statusLink = <Link to={appLink}>View filing status</Link>
+    var statusLink = <Link className="statusLink" to={appLink}>View filing status</Link>
```

Comment:
Yeah, I get it. I'm just more in favor of using pre-existing elements and styles rather than creating the one-offs. Something really clean about taking advantage of simple HTML.

A couple possible options:

1. __my recommendation__, ignore any CSS comments because we're going to go with the USDS stuff in #155 and I'll likely do some re-writing anyway
1. if you want ignore option one 😄 , rename the class to something more re-usable ... this is something I've been thinking about across the project
  - the component based CSS has advantages in some cases, but limits re-use in most (its why i created #116)
  - i think we need to do this only in very specific cases, more to come in #155